metadata.json

```diff
@@ -24,12 +24,10 @@
       "version_requirement": ">= 0.6.2 < 1.0.0"
     }
   ],
-  "data_provider": null,
   "requirements": [
     {
       "name": "puppet",
       "version_requirement": ">= 4.0.0"
```

Comment:
Does this still work in puppet 4?

Reply:
I don't know...I didn't test it. The `:undef` vs `nil` change brought in by ruby 2.5.1 might be enough to break it though, so probably not.